File: 10 - Python/Exercícios/02 - Estrutura Condicional/Exercício 10.py
#Exercício - Múltiplos

print("Digite dois números inteiros: ")
a = int(input(""))
b = int(input(""))

if a % b == 0 or b % a == 0:
    print("Os números são múltiplos.")
else:
    print("Os números não são múltiplos.")

File: 07 - C++/Exercícios/02 - Estrutura Condicional/Exercício 10.cpp
//Exercício - Multiplos

#include <iostream>

using namespace std;

int main() {

    int a, b;

    cout << "Digite dois numeros inteiros:\n";
    cin >> a;
    cin >> b;

    if (a % b == 0 || b % a == 0){
        cout << "Sao multiplos";
    }
    else{
        cout << "Nao sao multiplos";
    }

    return 0;

}